### PR TITLE
Quix Samples Library Release

### PR DESCRIPTION
**Pull request by SteveRosam**
Several samples updated to use Quix Streams

### CODE REVIEW

**Comment by SteveRosam** [comment]
Needs to go at the same time as https://github.com/quixio/quix-docs/pull/100
Approve but don't merge please

File: python/transformations/Hugging-Face-Model/hugging_face_model.py [hunk 1]
```diff
@@ -22,16 +22,16 @@ def on_event_data_handler(self, data: EventData):
...
             .add_value("label", label) \
             .add_value("sentiment", score) \
-            .write()
+            .add_tags(data.tags) \
+            .publish()
 
-    # Callback triggered for each new table data.
-    def on_parameter_data_handler(self, data: ParameterData):
+    # triggered for each new timeseries data.
+    def on_parameter_data_handler(self, stream_consumer: qx.StreamConsumer, data: qx.TimeseriesData):
```
Comment (tbedford) [inline python/transformations/Hugging-Face-Model/hugging_face_model.py:34]: Now that we've moved from parameter data to time-series data, should this function name be changed too?

File: python/transformations/InMemory-View/library.json [hunk 2]
```diff
@@ -8,8 +8,8 @@
         "Type": ["Code samples"],
         "Pipeline Stage": ["Transformation"]
     },
-    "shortDescription": "Template of how to build in memory view.",
-    "DefaultFile": "quix_function.py",
+    "shortDescription": "A template demonstrating how to build an in memory view.",
```
Comment (tbedford) [inline python/transformations/InMemory-View/library.json:11]: "build an in memory view." -> in-memory

**Review by tbedford — APPROVED** [review]
Minor comments, but not blockers, so approving.

File: python/transformations/Event-Detection/main.py [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
-# Quix injects credentials automatically to the client. Alternatively, you can always pass an SDK token manually as an argument.
-client = QuixStreamingClient()
+# Quix injects credentials automatically to the client.
+# Alternatively, you can always pass an SDK token manually as an argument.
+client = qx.QuixStreamingClient()
 
 print("Opening input and output topics")
-input_topic = client.open_input_topic(os.environ["input"], "default-consumer-group")
-output_topic = client.open_output_topic(os.environ["output"])
+consumer_topic = client.get_topic_consumer(os.environ["input"], "default-consumer-group")
```
[the remark this answers is not in the document]
Comment (SteveRosam) [reply python/transformations/Event-Detection/main.py:11]: doesnt have to be

**Review by tbedford — APPROVED** [review]

**Pull request merged**